chain/store.go

```diff
@@ -30,19 +30,30 @@ var logStore = logging.Logger("chain.store")
 
 var headKey = datastore.NewKey("/chain/heaviestTipSet")
 
+type globalIPLDSource struct {
```

Comment:
I don't like the name "global" here. The chain store shouldn't be making broad assertions outside its function. Better could be just `ipldSource`, `chainAndStateSource`.

Reply:
I understand where you're coming from.  Naming it like this came from an impulse to identify an important and too easy to misunderstand property of the ipld store.  I'll change it back though.